include/Reader/readerir.h

```diff
@@ -1412,12 +1412,13 @@ class GenIR : public ReaderBase {
                                unsigned int NumReservedValues,
                                const llvm::Twine &NameStr);
 
-  /// Add a new operand to the PHI instruction. The type of the new operand may
-  /// or may not equal to the type of the PHI instruction. Adjust the types as
-  /// necessary.
+  /// Update all undef placeholders corresponding to the new operand in the
+  /// PHI instruction. The type of the new operand may or may not equal to the
```

Comment:
pre-existing typo: should be "may or may not be equal to the ..." or "may or may not equal the ..."